src/pages/ContentfulPage.vue

```diff
@@ -160,6 +161,21 @@ const getContentGroups = pageData => {
 	})).filter(group => typeof group.component === 'function');
 };
 
+const componentOptions = {
```

Comment:
While this is fine to start with, we might consider moving this to a utility file or mixin at some point...

Reply:
Potentially, even component specific...

Reply:
Yeah I think this could also be passed as a prop to ContentfulPage, and then it can be defined by whatever page is actually using it.